src/docs/community/our-team.mdx

```diff
@@ -2,19 +2,19 @@ import { Divider, MaintainersCards } from '../../app/components';
 
--   The seeds of PX Blue were sown in **2011** as a single project, Power Xpert Insight.
+-   The seeds of Brightlayer UI were sown in **2011** as a single project, Power Xpert Insight.
 -   In **2013**, work on the Power Xpert Insight Gateway exposed a need for a shared design system between similar products.
--   The first iteration of PX Blue was a component library built in Polymer in **2015**.
--   In the summer of **2017**, difficulties with the Polymer component library led the team to conduct a full-scale design thinking project to rethink PX Blue. The team conducted extensive interviews with designers and developers to understand what they wanted out of a design system.
--   At the end of **2017**, the team evaluated existing open-source design systems and selected Material Design to serve as the foundation of the new PX Blue.
--   In February of **2018**, PX Blue 2.0 launches with support for Angular and React.
--   In September **2018**, PX Blue goes open source and makes all of the resources and materials available to the world. We also added support for developing mobile applications in addition to web.
--   In **2019**, PX Blue ramped up development of custom component libraries to supplement the components available from Material Design.
--   In **2020**, the team released an updated CLI for automatically starting new projects with PX Blue integration and began working on new multi-screen re-usable workflows. The team also won [two awards](https://www.designinnovationglobal.com/design-thinking/articles/di-global-awards-winners): Brand transformation and Enterprise Design Transformation.
+-   The first iteration of Brightlayer UI was a component library built in Polymer in **2015**.
+-   In the summer of **2017**, difficulties with the Polymer component library led the team to conduct a full-scale design thinking project to rethink Brightlayer UI. The team conducted extensive interviews with designers and developers to understand what they wanted out of a design system.
+-   At the end of **2017**, the team evaluated existing open-source design systems and selected Material Design to serve as the foundation of the new Brightlayer UI.
+-   In February of **2018**, Brightlayer UI 2.0 launches with support for Angular and React.
+-   In September **2018**, Brightlayer UI goes open source and makes all of the resources and materials available to the world. We also added support for developing mobile applications in addition to web.
+-   In **2019**, Brightlayer UI ramped up development of custom component libraries to supplement the components available from Material Design.
+-   In **2020**, the team released an updated CLI for automatically starting new projects with Brightlayer UI integration and began working on new multi-screen re-usable workflows. The team also won [two awards](https://www.designinnovationglobal.com/design-thinking/articles/di-global-awards-winners): Brand transformation and Enterprise Design Transformation.
```

Comment:
Would like to see some mentioning of the old name "PX Blue" and its renaming to "Brightlayer UI" somewhere in this section

Reply:
```suggestion
-   In **2020**, the team released an updated CLI for automatically starting new projects with PX Blue integration and began working on new multi-screen re-usable workflows. The team also won [two awards](https://www.designinnovationglobal.com/design-thinking/articles/di-global-awards-winners): Brand transformation and Enterprise Design Transformation.
```